test/Microsoft.ML.TestFramework/Attributes/BenchmarkTheoryAttribute.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
+using Microsoft.ML.TestFrameworkCommon.Attributes;
```

Comment:
Is this using still necessary?

Reply:
Dont think so. I have removed it and i'll build locally before I push that change.